spec/requests/api/projects_spec.rb

```diff
@@ -119,57 +119,33 @@
 
   describe 'POST /projects' do
     context 'maximum number of projects reached' do
-      before do
-        (1..user2.projects_limit).each do |project|
-          post api('/projects', user2), name: "foo#{project}"
-        end
-      end
-
-      it 'should not create new project' do
+      it 'should not create new project and respond with 403' do
+        User.any_instance.stub(:projects_limit_left).and_return(0)
         expect {
           post api('/projects', user2), name: 'foo'
         }.to change {Project.count}.by(0)
+        response.status.should == 403
       end
     end
 
-    it 'should create new project without path' do
-      expect { post api('/projects', user), name: 'foo' }.to change {Project.count}.by(1)
-    end
-
-    it 'should not create new project without name' do
-      expect { post api('/projects', user) }.to_not change {Project.count}
-    end
-
-    it 'should return a 400 error if name not given' do
-      post api('/projects', user)
-      response.status.should == 400
+    it 'should create new project without path and return 201' do
+      expect { post api('/projects', user), name: 'foo' }.
+        to change { Project.count }.by(1)
+      response.status.should == 201
     end
 
     it 'should create last project before reaching project limit' do
-      (1..user2.projects_limit-1).each { |p| post api('/projects', user2), name: "foo#{p}" }
+      User.any_instance.stub(:projects_limit_left).and_return(1)
       post api('/projects', user2), name: 'foo'
       response.status.should == 201
     end
 
-    it 'should respond with 201 on success' do
-      post api('/projects', user), name: 'foo'
-      response.status.should == 201
-    end
-
-    it 'should respond with 400 if name is not given' do
-      post api('/projects', user)
+    it 'should not create new project without name and return 400' do
+      expect { post api('/projects', user) }.to_not change { Project.count }
       response.status.should == 400
     end
 
-    it 'should return a 403 error if project limit reached' do
-      (1..user.projects_limit).each do |p|
-        post api('/projects', user), name: "foo#{p}"
-      end
-      post api('/projects', user), name: 'bar'
-      response.status.should == 403
-    end
-
-    it 'should assign attributes to project' do
+    it "should assign attributes to project" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.